src/components/common/JoinUpdate.js

```diff
@@ -0,0 +1,168 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Row, Col, Typography } from 'antd';
+const { Title } = Typography;
+const initialForm = {
+  'First Name': '',
+  'Last Name': '',
+  'Email Address': '',
+  Role: 'Parent',
+  Notes: '',
+};
+const Form = props => {
+  const [value, setValues] = useState(initialForm);
+
+  const onChange = event => {
+    const name = event.target.name;
+    setValues({ ...value, [name]: event.target.value });
+  };
+  const onSubmit = e => {
+    e.preventDefault();
+    axios
+      .post(
+        'https://api.airtable.com/v0/app4rHdRcowslAwKp/Table%201',
```

Comment:
This is all looking great, just one question: Why are we pushing this data to Airtable? Who's Airtable is this going to?

Reply:
Not sure of why airtable... but This is going to John's airtable, he stated he would have to get with Brianna and make her an account.

Reply:
This is an Airtable created by Brianne, as it was her preferred location to receive requests for information over the use of sending forms to email. I actually realized later that I only had an API key I created for this function. I'll have to make another task to change the key to hers, once I'm able to retrieve that from her.